frontend/src/components/object/ObjectTable.vue

```diff
@@ -85,18 +96,47 @@ async function showPermissions(objectId: string) {
 
 watch(getObjects, async () => {
   clearSelected();
+  loadLazyData(lazyParams);
```

Comment:
i dont think passing `lazyParams` to this function is doing anything

Reply:
I made it optional so we don't need to pass everytime.